aspnetcore/aspnetcore-2.0.md

```diff
@@ -74,11 +74,176 @@ For information about the status of planned documentation, see the [GitHub issue
+                });
+```
+
+The ASP.NET Core 2.0 project templates set up configuration and logging as shown above by default. If you only need `Console` and `Debug` logging and a `Logging` configuration section to control filtering, you don't need to add any extra code. However, you won't see the preceding code in your projects because the project template code calls a `CreateDefaultBuilder` method to do all that.  Here's the project template code with an extra `ConfigureLogging` call that shows how to add a provider:
+
+```csharp
+WebHost.CreateDefaultBuilder(args)
+    .UseStartup<Startup>()
+    .ConfigureLogging(logging => logging.AddEventLog())
+    .Build();
+```
+
+### Configuring filtering
+
+In ASP.NET Core 1.x, it's difficult to configure filtering: the way to do it across providers is not intuitive, and if you do it for individual providers, each provider handles it differently. In 2.0 we have changed the default `ILoggerFactory` to support filtering in a way that lets you use one flexible approach for both cross-provider filtering and specific-provider filtering.  This one approach is designed to work with `IConfiguration`, so you can control filtering from whatever configuration sources you choose. 
+
+### Create filter rules in configuration
+
+The `CreateDefaultBuilder` method sets up logging to look for configuration in a `Logging` section. The configuration data specifies minimum log levels by provider and category, as in the following example:
+
+```json
+{
+  "Logging": {
+    "IncludeScopes": false,
+    "Debug": {
+      "LogLevel": {
+        "Default": "Warning"
+      }
+    },
+    "Console": {
+      "LogLevel": {
+        "Microsoft.AspNetCore.Mvc.Razor.Internal": "Warning",
+        "Microsoft.AspNetCore.Mvc.Razor.Razor": "Debug",
+        "Microsoft.AspNetCore.Mvc.Razor": "Error",
+        "Default": "Warning"
+      }
+    },
+    "": {
+      "LogLevel": {
+        "Default": "Debug"
+      }
+    }
+  }
+}
+
+```
+
+This JSON creates six filter rules, one for the Debug provider, four for the Console provider, and one that applies to all providers. You'll see [later](#how-filtering-rules-are-applied) how a single rule is chosen for each provider when an `ILogger` object is created.
+
+### Filter rules in code
+
+You can register filter rules in code, as shown in the following example:
+
+```csharp
+WebHost.CreateDefaultBuilder(args)
+    .UseStartup<Startup>()
+    .ConfigureLogging(logging => logging.AddFilter("System", LogLevel.Debug))
+    .ConfigureLogging(logging => logging.AddFilter<DebugLoggerProvider>("Microsoft", LogLevel.Trace))
+    .Build();
+```
+
+The first `AddFilter` call applies to all providers. The second specifies the Debug provider by using its type name.
+
+### How filtering rules are applied
+
+The configuration data and the `AddFilter` code shown create the following rules. The first six come from the configuration example and the last two come from the code example:
```

Comment:
can you add anchor? `[configuration example](#config) and [code example](#codeEx)` ?